sql/src/main/java/org/apache/druid/sql/calcite/planner/CalciteRulesManager.java

```diff
@@ -82,6 +83,10 @@ public class CalciteRulesManager
   private static final int HEP_DEFAULT_MATCH_LIMIT = Integer.parseInt(
       System.getProperty(HEP_DEFAULT_MATCH_LIMIT_CONFIG_STRING, "1200")
   );
+  private static final String BLOAT_PROPERTY = "druid.sql.planner.bloat";
+  private static final int BLOAT = Integer.parseInt(
```

Comment:
It should be passed via context parameter or through runtime property. A context param seems a better fit as it would allow you to run those complex queries without having any impact on the rest of the queries.

Reply:
Good point, thanks!  I will have a look

Reply:
Updated to use context parameter